docs/index.md

```diff
@@ -62,13 +62,23 @@ TAU Profiles (mobile, TV, wearable) have differences, so some of the examples wi
 ## Where to get it
 
 The framework is included on all Tizen devices and the SDK emulator (starting from Tizen 2.3). Just create
-a new web application in the Tizen SDK IDE.
+a new web application in the Tizen SDK IDE. See also our [documentation](https://developer.tizen.org/dev-guide/5.0.0/org.tizen.web.apireference/html/ui_fw_api/ui_fw_api_cover.htm) and [github repository](https://github.com/Samsung/TAU).
```

Comment:
documentation --> API documentation
why link is more specific not just https://developer.tizen.org/development/api-references/web-application